package.json

```diff
@@ -17,7 +17,8 @@
     "docs:clone": "[ -d ./docs ] || git clone --branch gh-pages git@github.com:unstoppabledomains/namicorn.git docs",
     "docs:generate": "yarn run typedoc --out \"./docs/v$npm_package_version\"",
     "docs:readme": "./docs-readme.sh",
-    "docs:deploy": "./deploy-docs.sh"
+    "docs:deploy": "./deploy-docs.sh",
+    "git:push-tag": "git push origin v$npm_package_version"
```

Comment:
I think it is more logical to name this task `release:tag`.

Reply:
@bogdan done